### PR TITLE
[DONOTMERGE] CustomerBalanceTransaction support

### PR DESCRIPTION
**Pull request by remi-stripe**
This is a WIP PR to help review the other PRs and see fields added on other resources too.

cc @ob-stripe @mickjermsurawong-stripe

### CODE REVIEW

File: src/main/java/com/stripe/param/CustomerTransactionsParams.java [hunk 1]
```diff
@@ -0,0 +1,124 @@
+// Generated by com.stripe.generator.entity.SdkBuilder
+
+package com.stripe.param;
+
+import com.google.gson.annotations.SerializedName;
+import com.stripe.net.ApiRequestParams;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CustomerTransactionsParams extends ApiRequestParams {
```
Comment (remi-stripe) [inline src/main/java/com/stripe/param/CustomerTransactionsParams.java]: Hmmm this class is not named properly I thiiiiink. Not yet sure how to fix it.

Comment (mickjermsurawong-stripe) [reply src/main/java/com/stripe/param/CustomerTransactionsParams.java]: remi, what's the expected param class name do you have in mind?

Currently, if we name method of listing of customer balance transactions as `transactions`, the param class has it as  Customer (resource name) + Transactions (method name) + Params.
```
public CustomerBalanceTransactionCollection transactions(CustomerTransactionsParams params) {}
```

The similar example here is `persons` method on `Account`
```
public PersonCollection persons(AccountPersonsParams params) {}
```

Comment (remi-stripe) [reply src/main/java/com/stripe/param/CustomerTransactionsParams.java]: I expected `TransactionParams` in that world I think

Comment (mickjermsurawong-stripe) [reply src/main/java/com/stripe/param/CustomerTransactionsParams.java]: I think having resource prefixing (Customer) is necessary to have consistent namespace for the params on a given resource. 
Depending on the final name we agreed, I would be more inclined to having `Customer` + method name still.

**Comment by remi-stripe** [comment]
Closing as we have released support.

**Pull request closed without merging**